### PR TITLE
[JUJU-4479] Add context.Context in client/(i-p)* facades

### PR DESCRIPTION
**Pull request by anvial**
This PR adds context.Context parameter to:
- client/imagemetadatamanager
- client/keymanager
- client/machinemanager
- client/metricsdebug
- client/modelconfig
- client/modelgeneration
- client/modelmanager
- client/modelupgrader
- client/payloads
- client/pinger 

facade calls and fixes the unit tests.

## Checklist

- [x] Code style: imports ordered, good names, simple structure, etc
- [x] Comments saying why design decisions were made
- [x] Go unit tests, with comments saying what you're testing
- [ ] ~[Integration tests](https://github.com/juju/juju/tree/main/tests), with comments saying what you're testing~
- [ ] ~[doc.go](https://discourse.charmhub.io/t/readme-in-packages/451) added or updated in changed packages~

## QA steps

```sh
go test github.com/juju/juju/apiserver/facades/client/imagemetadatamanager/... -gocheck.v 
go test github.com/juju/juju/apiserver/facades/client/keymanager/... -gocheck.v 
go test github.com/juju/juju/apiserver/facades/client/machinemanager/... -gocheck.v 
go test github.com/juju/juju/apiserver/facades/client/metricsdebug/... -gocheck.v 
go test github.com/juju/juju/apiserver/facades/client/modelconfig/... -gocheck.v 
go test github.com/juju/juju/apiserver/facades/client/modelgeneration/... -gocheck.v 
go test github.com/juju/juju/apiserver/facades/client/modelmanager/... -gocheck.v 
go test github.com/juju/juju/apiserver/facades/client/modelupgrader/... -gocheck.v 
go test github.com/juju/juju/apiserver/facades/client/payloads/... -gocheck.v 
go test github.com/juju/juju/apiserver/facades/client/pinger/... -gocheck.v 

make build && make install && juju version
juju bootstrap lxd lxd
```

### CODE REVIEW

File: apiserver/facades/client/modelmanager/modelmanager.go [hunk 1]
```diff
@@ -5,7 +5,6 @@ package modelmanager
 
 import (
 	"context"
-	stdcontext "context"
```
Comment (nvinuesa) [inline apiserver/facades/client/modelmanager/modelmanager.go:8]: why is this import removed? only the name `stdcontext` should be removed but not the import, right?

Comment (anvial) [reply apiserver/facades/client/modelmanager/modelmanager.go:8]: looks like , I didnot understand you, 

I've removed `stdcontext`, and used `context` instead.

Comment (nvinuesa) [reply apiserver/facades/client/modelmanager/modelmanager.go:8]: oh sorry, indeed you kept the previous import. Sorry for the noise

**Review by nvinuesa — APPROVED** [review]
LGTM, QA passes
